app/uk/gov/hmrc/fhregistrationfrontend/views/forms/contact_person.scala.html

```diff
@@ -41,8 +41,8 @@
 @main_template(title = titlePrefix + title, scriptElem = Some(pageScripts)) {
 
   <link rel="stylesheet" href='@routes.Assets.at("stylesheets/fh.css")'/>
-
   @Back(navigation)
+  <a href="@uk.gov.hmrc.fhregistrationfrontend.controllers.routes.EmailVerificationController.submitContactEmail()" class="link-back">@Messages("fh.generic.back")</a>
```

Comment:
can we check if the email is already verified if is any point to send them back to email verification. We might need for now to keep just the back link on business type page